pkgs/development/python-modules/pyscf/default.nix

```diff
@@ -1,21 +1,26 @@
-{ buildPythonPackage, lib, fetchFromGitHub, libcint, libxc, xcfun, blas
-, numpy, scipy, h5py
+{ buildPythonPackage, python3, lib, fetchFromGitHub, libcint, libxc, xcfun, blas, cmake
+, numpy, scipy, h5py, cppe, nose, nose-exclude
 }:
 
 buildPythonPackage rec {
   pname = "pyscf";
-  version = "1.7.6.post1";
+  version = "2.0.0";
 
   src = fetchFromGitHub {
     owner = "pyscf";
     repo = pname;
-    rev = "f6c9c6654dd9609c5e467a1edd5c2c076f793acc";
-    sha256  = "0xbwkjxxysfpqz72qn6n4a0zr2h6sprbcal8j7kzymh7swjy117w";
+    rev = "v${version}";
+    sha256  = "J6t/Oq7WE5wz2aefok5c47CeiP2DskLF//JWGNzgRpw=";
```

Comment:
This looks like a base64-encoded sha256. Would probably be worth changing to the SRI rep: ``hash = "sha256-..."`